documentation.md

```diff
@@ -0,0 +1,86 @@
+---
+layout: ballerina-learn-landing-page
+title: Ballerina Documentation
+description: Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below.
+keywords: ballerina, learn, documentation, docs, programming language
+permalink: /learn-beta2/
+intro: Ballerina is a comprehensive language that is easy to grasp for anyone with prior programming experience. Start learning with the material below.
+redirect_from:
+- /learn-beta2
+---
+## Getting Started
+
+### [Language Introduction Video](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)
+
+Video presentation of above which people can use as a sample of how to present the one hour deck.
+
+### [Language Taste Screencasts](https://www.youtube.com/watch?v=My_uqtHvXV8&t=10s)
+
+End-to-end video screencasts like the one Manu has done on writing a service.
```

Comment:
End-to-end screencasts covering popular use cases that give a taste of the Ballerina language.

Reply:
Fixed.